src/Array.ts

```diff
@@ -463,7 +465,7 @@ const chunk =
 
         const numChunks = Math.ceil(as.length / chunkSize)
 
-        const chunks: A[][] = [...globalThis.Array(numChunks)].map(() => [])
+        const chunks: A[][] = globalThis.Array(numChunks).map(() => [])
```

Comment:
Any reason this wouldn't be safe? If we're creating a fresh array we don't need to clone it, right? (One of the rules complains about spreading a `any[]`)

Reply:
If the tests pass, I'm down. I think I was just copying some kind of spread technique from stack overflow 😁

Reply:
So it does the break the tests.. but I don't entirely understand why :thinking:

Reply:
If you do `Array(3)` and `[...Array(3)]` in your browser console, I think you can see the difference. I'm not sure why it behaves that way 🤷‍♂️

Reply:
tl;dr: JavaScript is dumb. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#array_methods_and_empty_slots